website/docs/reference/project-collaboration-mode.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Project collaboration mode
+---
+
+:::info Availability
+
+The change requests feature is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+
+:::
+
+Unleash project collaboration mode is a feature that allows to specify who can submit [change requests](change-requests.md). Currently, there are two modes:
+- **open** where everyone can submit change requests. This is a default setting for the existing and new Unleash projects.
+- **protected** where [admins, project members and owners](rbac.md#standard-roles) can submit change requests. Users who are not added to projects will not be able to submit change requests.
```

Comment:
Oh, for context: maybe we can explode this out into two separate sections. That makes it easier to explain each mode in more depth (using multiple paragraphs)

Reply:
Makes sense. Picking and applying your changes. With one minor change. Instead of "Only [admins, project members and owners]" we'll say "only admins and project members"